React/Views/RCTModalHostView.m

```diff
@@ -54,6 +61,27 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
   return self;
 }
 
+#if TARGET_OS_TV
+- (void)menuButtonPressed:(__unused UIGestureRecognizer *)r
+{
+    if(_onRequestClose) {
+        _onRequestClose(nil);
+    }
+}
+
+- (void)setOnRequestClose:(RCTDirectEventBlock)onRequestClose
+{
+  _onRequestClose = onRequestClose;
+  if([self.subviews count] > 0) {
+    if(_onRequestClose && _tapGestureRecognizer) {
+      [self.subviews[0] addGestureRecognizer:_tapGestureRecognizer];
```

Comment:
This class seems to be set up so that `subviews` has either 0 or 1 views in it.  I can use `_reactSubview` instead, I suppose....